README.md

```diff
@@ -405,7 +405,7 @@ FormBuilder allows you to add skip logic and functional support to field to enha
 "skipLogic": {
   "checks": [{
     "variable": "form.bmi.$modelValue", // variable to check, can be a FormBuilder global variable as well (e.g. global.gender)
-    "operand": ">", // operand to perform ( "=", "!=", "<", "<=", ">", ">=" )
+    "operand": ">", // operand to perform ( "=", "!=", "<", "<=", ">", ">=", "[in]", "[!in]" )
```

Comment:
why not call it in or !in/notin instead of [in] and [!in]?

Reply:
Reason for the brackets is to indicate that this is for an Array only. I was thinking of adding another for String only but was a bit more work and not needed yet. In future we can add one for string that will read "in" "!in"

Reply:
i think it makes sense for the same to receive either a string or array, it can do a datatype or index check to treat either as array or string

Reply:
ok cool thanks Kaweesi that makes sense. Ive removed the bracket which will now allow an array or string to be checked for the indexOf a value